### PR TITLE
Mistral 7B macOS M1/M2

### PR DESCRIPTION
**Pull request by casperdcl**
Follow-up to https://github.com/premAI-io/prem-services/pull/117

- instruct
  + [x] manifest
  + [x] readme
- openorca
  + [x] manifest
  + [x] readme

### CODE REVIEW

File: chat-mistral-7b-instruct/manifest.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "id": "mistral-7b-instruct",
+    "name": "Mistral 7B Instruct",
+    "beta": true,
+    "description": "",
+    "documentation": "",
+    "serviceType": "binary",
+    "version": "1",
+    "icon": "",
+    "modelInfo": {
+        "memoryRequirements": 4096
```
Comment (casperdcl) [inline chat-mistral-7b-instruct/manifest.json:11]: btw I didn't check this - please do verify actual number

File: chat-mistral-7b-instruct/manifest.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    "interfaces": [
+        "chat"
+    ],
+    "defaultPort": 8000,
+    "defaultExternalPort": 8448,
```
Comment (casperdcl) [inline chat-mistral-7b-instruct/manifest.json:17]: what's the convention here? fyi for other existing models:

- chat-falcon-7b-instruct: 8448
- chat-mpt-7b-instruct: 8452

Comment (filopedraz) [reply chat-mistral-7b-instruct/manifest.json:17]: Some random numbers. No real convention

File: chat-mistral-7b-instruct/manifest.json (hunk 1 above)
Comment (filopedraz) [reply chat-mistral-7b-instruct/manifest.json:11]: Sure, l ll do it after deployment

File: chat-mistral-7b-instruct/manifest.json (hunk 2 above)
Comment (casperdcl) [reply chat-mistral-7b-instruct/manifest.json:17]: we should address this in https://github.com/premAI-io/prem-services/issues/120 :)

**Comment by biswaroop1547** [comment]
I've tested both the models on M2 + updated readme with some generation results from them.
Latency was good and similar for both i.e ~8 tokens/sec

**Review by casperdcl — COMMENTED** [review]
lgtm - @biswaroop1547 please mark the PR as approved when ready :)

**Review by biswaroop1547 — APPROVED** [review]

**Comment by biswaroop1547** [comment]
lgtm, the `--port` change is dependent on https://github.com/premAI-io/prem-services/pull/122

**Comment by filopedraz** [comment]
I merge this one. We can have a small PR when the port fix is merged.

**Pull request merged**